test/metabase/pulse/render/body_test.clj

```diff
@@ -648,7 +643,7 @@
 
 (def donut-info #'body/donut-info)
 
-(deftest donut-info-test
+(deftest ^:parallel donut-info-test
```

Comment:
While I was going thru everything I was able to make a few more tests `:^parallel`. There are probably more in this namespace but I didn't check everything